preview-env/comment/templates/comment-footer.md

```diff
@@ -2,3 +2,6 @@
 The :link: `ArgoCD` link can be used to check the state and configuration of all the services deployed as part of the preview environments and get logs.
 
 In case of error, please check :clipboard: `Deployment Jobs` and :link: `ArgoCD` to debug, and check our (yet to come) troubleshooting page
+
+> [!NOTE]
+> There's a chance, that timing issues cause a deployment to be shown as failure here, but are actually still progressing or even became healthy in the meantime in ArgoCD. When in doubt: Just check both ArgoCD links!
```

Comment:
I think this note is a bit overwhelming for users at this point. I would remove it.

Reply:
especially when all deployment are successful.

Reply:
agree. Glad I put it into a separate commit. Removed it now :-)